.analysis_options

```diff
@@ -1,30 +1,68 @@
 analyzer:
   strong-mode: true
   language:
-      enableSuperMixins: true
+    enableStrictCallChecks: true
+    enableSuperMixins: true
+  errors:
+    # Allow having TODOs in the code
+    todo: ignore
+
 linter:
   rules:
-    # Errors
+    # these rules are documented on and in the same order as
+    # the Dart Lint rules page to make maintenance easier
+    # http://dart-lang.github.io/linter/lints/
+
+    # === error rules ===
     - avoid_empty_else
+    # TODO - comment_references
+    - cancel_subscriptions
+    # TODO - close_sinks
     - control_flow_in_finally
     - empty_statements
+    - hash_and_equals
+    - invariant_booleans
+    - iterable_contains_unrelated_type
+    - list_remove_unrelated_type
+    - literal_only_boolean_expressions
     - test_types_in_equals
     - throw_in_finally
+    - unrelated_type_equality_checks
     - valid_regexps
 
-    # Style
-    # TODO: - annotate_overrides
+    # === style rules ===
+    - always_declare_return_types
+    # TODO - always_specify_types
+    # TODO - annotate_overrides
+    # TODO - avoid_as
     - avoid_init_to_null
     - avoid_return_types_on_setters
     - await_only_futures
-    - camel_case_types
-    # TODO: - comment_references
+    # TODO - camel_case_types
+    # TODO - constant_identifier_names
```

Comment:
if you want flutter style, this should be on, unless you have good reason (in which case maybe just //ignore the good reasons)